HIP/hip-0000-mirror-node-account-id.md

```diff
@@ -0,0 +1,68 @@
+- hip: <HIP number (this is determined by the HIP editor)>
```

Comment:
Do you mind aligning it with the new HIP-1 and the new https://github.com/hashgraph/hedera-improvement-proposal/blob/master/hip-0000-template.md template?

The header should be reformatted to align with the table structure (remove `-` at each line in the header and wrap the header with `---` on top and bottom.

Please also make note that the review process has changed as well.